test/e2e/specs/plugins/plugins__search.ts

```diff
@@ -20,10 +20,39 @@ describe( DataHelper.createSuiteTitle( 'Plugins search' ), function () {
 	it( 'Visit plugins page', async function () {
 		pluginsPage = new PluginsPage( page );
 		await pluginsPage.visit();
+		if ( envVariables.VIEWPORT_NAME !== 'mobile' ) {
+			// Ensure the page is wide enough to show the breadcrumb details.
+			await page.setViewportSize( { width: 1300, height: 1080 } );
+		}
 	} );
 
-	it( 'Search for ecommerce', async function () {
-		await pluginsPage.search( 'ecommerce' );
-		await pluginsPage.validateExpectedSearchResultFound( 'WooCommerce' );
+	it( 'Search for shipping', async function () {
+		await pluginsPage.search( 'shipping' );
+		await pluginsPage.validateExpectedSearchResultFound( 'Royal Mail' );
+	} );
+
+	it( 'Can click on a search result', async function () {
```

Comment:
**issue**: test step names should follow style guide outlined [here](https://wpcalypso.wordpress.com/devdocs/test/e2e/docs/style_guide.md#do-not-use-modal-verbs).

Reply:
Ah, thanks for the link!